backend/src/models/index.ts

```diff
@@ -1,5 +1,5 @@
 import {Sequelize} from 'sequelize-typescript';
-import db from 'src/config/database';
+import db from '../config/database';
```

Comment:
It doesn't work with absolute paths. I am getting the following error when using `'src/config/database'`:

```js
Error: Cannot find module 'src/config/database'
    at Function.Module._resolveFilename (module.js:470:15)
    at Function.Module._load (module.js:418:25)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (D:\dev\projects\welovecoding\welovecoding\backend\dist\models\index.js:4:20)
```

Steps to reproduce:

```bash
cd backend
tsc
node dist/index.js
```